metaflow/plugins/kfp/kfp.py

```diff
@@ -132,11 +133,14 @@ def __init__(
         self.notify_on_error = notify_on_error
         self.notify_on_success = notify_on_success
 
-        self._client = kfp.Client(namespace=api_namespace, userid=username, **kwargs)
+        # kfp userid needs to have the user domain
+        kfp_client_user_email = username + KFP_USER_DOMAIN
```

Comment:
what would the value of `KFP_USER_DOMAIN` be? `@zillow.com`? Should that be documented?  Because `@domain` is not just "user_domain".

What if username is not provided, for the default setup of KFP?  Could we pass in `None`?

thanks!

Reply:
it would be @zillowgroup.com

we always have to have a username, it won't work otherwise. we need to pass in something to the kfp client

Reply:
technically, the domain name part is after the `@`.

`local-part@domain` see https://en.wikipedia.org/wiki/Email_address

Reply:
fixed